<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1669</width>
    <height>1467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1351</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 69, 125);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>861</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(255, 255, 255, 0));</string>
      </property>
      <property name="title">
       <string>Gestion Des Sponsors</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Nom De L'Entreprise </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Matricule Fiscale</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Adresse Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ad">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>70</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>310</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 186, 61);</string>
       </property>
       <property name="text">
        <string>+ Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_matri">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>70</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>80</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Adresse </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_email">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>210</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supp">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>310</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 186, 61);</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/new/boutons/supp2.jpg</normaloff>:/new/boutons/supp2.jpg</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>150</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Durée De La Publicité</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>220</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Montant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_matrisupp">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>310</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="te_duree">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>150</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="ds_montant">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>220</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(225, 225, 225);
</string>
      </property>
      <property name="text">
       <string>Gestion Des Sponsors</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>520</y>
        <width>711</width>
        <height>451</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Liste Des Sponsors</string>
      </property>
      <widget class="QPushButton" name="pb_recher">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 170, 170);
border-color: rgb(138, 138, 138);
color: rgb(45, 45, 45)</string>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/new/boutons/recher1.png</normaloff>:/new/boutons/recher1.png</iconset>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>8</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(138, 138, 138);
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.186957 rgba(97, 97, 97, 30), stop:0.930435 rgba(150, 150, 150, 48))</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_tri">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>380</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(225, 225, 225)</string>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/new/boutons/tri2.png</normaloff>:/new/boutons/tri2.png</iconset>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>380</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(225, 225, 225)</string>
       </property>
       <item>
        <property name="text">
         <string> &lt;Choisir Une Option&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Matricule Fiscale</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Date Du Contrat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Montant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Durée De La Publicité</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nom De L'Entreprise</string>
        </property>
       </item>
      </widget>
      <widget class="QTableView" name="tabsponsor">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>611</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modif">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>50</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 186, 61);</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/new/boutons/modifier.png</normaloff>:/new/boutons/modifier.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>620</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Imprimer Un Contrat</string>
      </property>
      <widget class="QPushButton" name="pb_pdf">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(225, 225, 225)</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/new/boutons/pdf1.png</normaloff>:/new/boutons/pdf1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>34</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>810</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Envoyer Un Mail</string>
      </property>
      <widget class="QPushButton" name="pb_mail">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(225, 225, 225)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/new/boutons/mailing3.jpg</normaloff>:/new/boutons/mailing3.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>37</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_geo">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>310</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(225, 225, 225)</string>
      </property>
      <property name="text">
       <string>Geolocalisation</string>
      </property>
      <property name="icon">
       <iconset resource="ressourcesponso.qrc">
        <normaloff>:/new/prefix1_bouttons/loc2.png</normaloff>:/new/prefix1_bouttons/loc2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>23</width>
        <height>26</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_calend">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>150</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(225, 225, 225)</string>
      </property>
      <property name="text">
       <string>Calendrier</string>
      </property>
      <property name="icon">
       <iconset resource="ressourcesponso.qrc">
        <normaloff>:/new/prefix1_bouttons/cal2.jpg</normaloff>:/new/prefix1_bouttons/cal2.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1669</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
  <include location="ressourcesponso.qrc"/>
 </resources>
 <connections/>
</ui>
